assets/src/dashboard/app/views/myStories/header/index.js

```diff
@@ -44,13 +46,18 @@ import {
   ViewPropTypes,
 } from '../../../../utils/useStoryView';
 import { useDashboardResultsLabel } from '../../../../utils';
-import {
-  BodyViewOptions,
-  HeaderToggleButtonContainer,
-  PageHeading,
-} from '../../shared';
+import { BodyViewOptions, PageHeading } from '../../shared';
 import { useConfig } from '../../../config';
 
+const StyledPill = styled(Pill)`
```

Comment:
Seeing very faint lines near the edges before the focus box shadow. It may be an artifact of our box shadow too though 🤷 
![image](https://user-images.githubusercontent.com/22185279/109061610-1ab95600-76a4-11eb-8f46-86ce671f68b2.png)

Reply:
🤔  I'll make a lil bug ticket to track.

Reply:
https://github.com/google/web-stories-wp/issues/6537